amd_openvx_extensions/amd_rpp/source/tensor/Resize.cpp

```diff
@@ -0,0 +1,265 @@
+/*
+Copyright (c) 2019 - 2023 Advanced Micro Devices, Inc. All rights reserved.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+#include "internal_publishKernels.h"
+
+struct ResizeLocalData {
+    vxRppHandle * handle;
+    Rpp32u device_type;
+    RppPtr_t pSrc;
+    RppPtr_t pDst;
+    vx_uint32 *resize_h;
+    vx_uint32 *resize_w;
+    RpptDescPtr src_desc_ptr;
+    RpptDesc srcDesc;
+    RpptDesc dstDesc;
+    RpptDescPtr dst_desc_ptr;
+    void *roi_tensor_ptr;
+    RpptROI * roi_ptr;
+    RpptRoiType roiType;
+    Rpp32s input_layout;
+    Rpp32s output_layout;
+    size_t in_tensor_dims[VX_TENSOR_DIMS];
+    size_t out_tensor_dims[VX_TENSOR_DIMS];
+    vx_enum in_tensor_type;
+    vx_enum out_tensor_type;
+    RpptImagePatch *dstImgSize;
+    Rpp32s interpolation_type; 
+};
+
+static vx_status VX_CALLBACK refreshResize(vx_node node, const vx_reference *parameters, vx_uint32 num, ResizeLocalData *data) {
+    vx_status status = VX_SUCCESS;
+    STATUS_ERROR_CHECK(vxCopyArrayRange((vx_array)parameters[3], 0, data->src_desc_ptr->n, sizeof(vx_uint32), data->resize_w, VX_READ_ONLY, VX_MEMORY_TYPE_HOST));
```

Comment:
This needs to be changed to inputDimensions[0] right instead of data->src_desc_ptr->n